.circleci/config.yml

```diff
@@ -17,6 +17,12 @@ jobs:
           <<: *cache-key
           paths:
             - ./node_modules
+      - run:
+          name: Install extension dependencies
+          command: npm run bootstrap
+      - run:
+          name: Check code style
+          command: npm run prettier-check
```

Comment:
I don't think it's needed on CI. The codebase is automatically prettified on commit and there's `lint` step to catch actual errors.

Reply:
I disagree. You can always bypass the the commit check (plus it's not even setup yet 😬), this way we make sure no one accidentally does weird things and master is always in a correct state.

The check is also super fast. With the handful of files currently in the repo it's < 1 second.